yt/yt/ytlib/scheduler/config.h

```diff
@@ -2189,6 +2191,7 @@ class TOperationFairShareTreeRuntimeParametersUpdate
     TJobResourcesConfigPtr ResourceLimits;
     // Can only be set by an administrator.
     std::optional<bool> EnableDetailedLogs;
+    std::optional<TBooleanFormula> SchedulingTagFilter;
```

Comment:
Do we actually need separate scheduling tag filter values per tree? I think it is not a good idea because this option affects the state of operation controller

Reply:
I will try removing it.